dartagnan/src/main/java/com/dat3m/dartagnan/configuration/Method.java

```diff
@@ -3,20 +3,16 @@
 import java.util.Arrays;
 
 public enum Method implements OptionInterface {
-	ASSUME, INCREMENTAL, TWO, CAAT;
+	EAGER, LAZY;
 	
 	// Used for options in the console
 	@Override
 	public String asStringOption() {
         switch(this) {
-        	case TWO:
-        		return "two";
-        	case INCREMENTAL:
-        		return "incremental";
-        	case ASSUME:
-        		return "assume";
-			case CAAT:
-				return "caat";
+        	case EAGER:
+        		return "eager";
+			case LAZY:
+				return "lazy";
```

Comment:
Can you double check the formatting? It looks wrong here (and in some other touched files).

Reply:
As far as I could see, it was just this file